todos-app/src/App/ListFooter/FilterList/FilterList.tsx

```diff
@@ -1,7 +1,55 @@
-import React from "react";
+import { FilterType } from "../../models/Filters";
+import "../../../App.css";
```

Comment:
I don't think you need this import..

Reply:
Are you talking about _import { FilterType }_?

Reply:
If so, then this is necessary to ensure the selection of filters and to ensure the _filter_ variable options can accept are limited.